packages/phone-number-privacy/signer/src/common/database/wrappers/request.ts

```diff
@@ -47,3 +47,27 @@ export async function insertRequest(
     await (trx != null ? sql.transacting(trx) : sql)
   })
 }
+
+export async function deleteRequestsOlderThan(
+  db: Knex,
+  date: Date,
+  logger: Logger,
+  trx?: Knex.Transaction
+): Promise<Number> {
+  logger.debug(`Removing request older than: ${date}`)
+  if (date > new Date()) {
+    logger.debug('Date is in the future')
+    return 0
+  }
+  return doMeteredSql(
+    'deleteRequestsOlderThan',
+    ErrorMessage.DATABASE_REMOVE_FAILURE,
+    logger,
+    async () => {
+      const sql = db<PnpSignRequestRecord>(REQUESTS_TABLE)
+        .where(REQUESTS_COLUMNS.timestamp, '<=', date)
+        .del()
```

Comment:
should we pass .timeout() here?

Reply:
As it could be a heavy query depending on the amount of rows to be delated, and as it's going to run as a background job, I thought that it was not necessary to do it